roles/repository/defaults/main.yml

```diff
@@ -107,3 +110,63 @@ repository_jdbc_driver_base_url: https://repo.maven.apache.org/maven2/org/postgr
 repository_jdbc_driver_url: >-
   {{ '{0}/{1}/postgresql-{1}.jar'.format(repository_jdbc_driver_base_url, repository_jdbc_driver_version) }}
 repository_jdbc_driver_checksum: "sha1:{{ repository_jdbc_driver_url }}.sha1"
+
+# Repository amps to install
+repository_amp_community_enabled: false
+
+repository_amp_alfresco_googledrive_repo_enterprise_enabled: "{{ 'false' if repository_amp_community_enabled else 'true' }}"
```

Comment:
I would avoid to define all of this here at role level, repository should only have a single argument with the list of amps to be fetched and installed, it doesn't really need anything else.

so the logic to build the list should be moved in the main playbook

Reply:
The logic we chose and you implemented for wars could be re-used as it allows for building the list of amps from the outer playbook and still have a sensible default list.
However there's something that needs to be taken into account: AMP can be applied either to alfresco or share wars.